packages/core/src/common/_variables.scss

```diff
@@ -24,13 +24,15 @@ $bp-ns: $ns;
 // (so other variables can use it to define themselves)
 $pt-grid-size: 10px !default;
 
+$bp-spacing: 4px !default;
```

Comment:
Why bp-spacing instead of pt-spacing? That creates inconsistency with all the other variables, and I'm not sure the prefix change is worth it.

Reply:
I'm not particularly fussed on the naming here, but I agree with wanting consistency. Changed in 00154e5203e5c8c414a538767bbc647f5fea136f.

I do think it's a bit odd to have `pt-*` prefixed variables in Blueprint considering that external users don't necessarily have any association with Palantir. When writing the variable name originally, I thought it might be a good way to differentiate between older and newer variables, but I agree consistency is more important for now.

Reply:
Sounds good. If we ever want to change the prefix, we should do it in a big bang (probably with one major version of having both) so that consumers can use them consistently.